### PR TITLE
pc-nrfutil: Fix PRN: only needs to check response from DFU Target whe…

### PR DESCRIPTION
**Pull request by qung**
…n reach PRN packet

### CODE REVIEW

File: nordicsemi/dfu/dfu_transport_ble.py [hunk 1]
```diff
@@ -343,16 +344,21 @@ def __select_object(self, object_type):
...
+        response = self.__get_response(DfuTransportBle.OP_CODE['CalcChecSum'])
 
+        (offset, crc) = struct.unpack('<II', bytearray(response))
+        return {'offset': offset, 'crc': crc}
 
     def __stream_data(self, data, crc=0, offset=0):
         logger.debug("BLE: Streaming Data: len:{0} offset:{1} crc:0x{2:08X}".format(len(data), offset, crc))
         def validate_crc():
-            
+
```
Comment (carlescufi) [inline nordicsemi/dfu/dfu_transport_ble.py]: line with whitespace, remove

File: nordicsemi/dfu/dfu_transport_ble.py [hunk 2]
```diff
@@ -343,16 +344,21 @@ def __select_object(self, object_type):
...
     def __stream_data(self, data, crc=0, offset=0):
         logger.debug("BLE: Streaming Data: len:{0} offset:{1} crc:0x{2:08X}".format(len(data), offset, crc))
         def validate_crc():
-            
+             
             if (crc != response['crc']):
                 raise ValidationException('Failed CRC validation.\n'\
                                 + 'Expected: {} Recieved: {}.'.format(crc, response['crc']))
-
+
```
Comment (carlescufi) [inline nordicsemi/dfu/dfu_transport_ble.py]: line with whitespace, remove

File: nordicsemi/dfu/dfu_transport_ble.py [hunk 3]
```diff
@@ -178,6 +178,7 @@ def __init__(self,
         self.target_device_addr = target_device_addr
         self.dfu_adapter        = None
         self.prn                = prn
+
```
Comment (carlescufi) [inline nordicsemi/dfu/dfu_transport_ble.py]: remove extra line

**Comment by carlescufi** [comment]
thanks!

**Comment by qung** [comment]
thanks Carles!

**Pull request merged**